pkg/app/middleware.go

```diff
@@ -61,3 +62,14 @@ func SetupCORSMiddleware(handler http.Handler, domains []string) http.Handler {
 	})
 	return corsHandler.Handler(handler)
 }
+
+func DisableV1Middleware(next http.Handler) http.Handler {
+	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
+		if strings.Contains(r.URL.Path, "/assisted-install/v1/") {
+			w.WriteHeader(http.StatusNotFound)
```

Comment:
@avishayt what do you think is the right error code to return here?

Reply:
Agree with this - it's a path that doesn't exist